src/content.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
+  res.render('content');
```

Comment:
so yeah here I would still pass the `english` and `arabic` variables

Reply:
Why?

Reply:
Well just in case you did ever want to change them, but it's fine this way, I just wanted to know why at one point you passed them as variables but then changed your mind.